Content.Shared/_RMC14/Xenonids/Stomp/SharedXenoStompSystem.cs

```diff
@@ -67,16 +70,16 @@ private void OnXenoStompAction(Entity<XenoStompComponent> xeno, ref XenoStompAct
                 SpawnAttachedTo(xeno.Comp.Effect, receiver.Owner.ToCoordinates());
         }
 
-        foreach (var receiver in _receivers)
+        foreach (var shortReceiver in _shortReceivers)
```

Comment:
For this I think it would be better to check standing state is down, I read your reasoning but it's already more likely to be able to hit multiple downed mobs here because of pixel movement, I think this would make it too strong